### ISSUE TITLE
userPopout issue

### ISSUE THREAD

**Issue by ciaoamigoschat** [rafaelalmeidatk/discord-react-clone#8]
When I click on a user at the bottom of the list user Pop Up it opens out of the area therefore not visible.

**Comment by rafaelalmeidatk** [comment]
Yeah this was one of the limitations with the popover written from scratch. Realistically I would use [Pooper](https://popper.js.org/) to manage the popover, because it handles this scenarios where they go out of the screen.

**Comment by ciaoamigoschat** [comment]
I liked your work, do you also work on commission? Where can I write to you?

**Comment by rafaelalmeidatk** [comment]
I am kinda busy at the moment but you may try to reach me by email, it is in my GitHub profile.
